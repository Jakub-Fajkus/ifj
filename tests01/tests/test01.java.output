  1. KEYWORD class
  2. ID Main
  3. {
  4. KEYWORD static
  5. KEYWORD void
  6. ID run
  7. (
  8. )
  9. {
 10. KEYWORD int
 11. ID a
 12. ;
 13. SPECIALID ifj16.print
 14. (
 15. LITERAL 'za@@dej' 
 16. )
 17. ;
 18. KEYWORD if
 19. (
 20. ID a
 21. <
 22. NUM 0
 23. )
 24. {
 25. }
 26. }
 27. }
 28. EOF
