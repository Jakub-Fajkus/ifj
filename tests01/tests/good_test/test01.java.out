Zadejte cislo pro vypocet faktorialu: Vysledek je: 5040
